src/pytorch_lightning/trainer/configuration_validator.py

```diff
@@ -115,9 +116,15 @@ def __verify_train_val_loop_configuration(trainer: "pl.Trainer", model: "pl.Ligh
     has_val_step = is_overridden("validation_step", model)
 
     if has_val_loader and not has_val_step:
-        rank_zero_warn("You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.")
+        rank_zero_warn(
+            "You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.",
```

Comment:
The warning below can be a false positive, but when would this one be?

Reply:
good point, it was the other warning that I was trying to suppress